packages/m/msgpack-cxx/xmake.lua

```diff
@@ -9,9 +8,16 @@ package("msgpack-cxx")
     add_versions("6.1.0", "23ede7e93c8efee343ad8c6514c28f3708207e5106af3b3e4969b3a9ed7039e7")
     add_versions("4.1.1", "8115c5edcf20bc1408c798a6bdaec16c1e52b1c34859d4982a0fb03300438f0b")
 
+    add_configs("header_only", {description = "Use header only version.", default = false, type = "boolean"})
```

Comment:
rename to headeronly

Reply:
Renaming the config to "headeronly" would make it inconsistent to the other packages which have this config name "header_only"